pkg/email/templates/added_as_collaborator_body.template

```diff
@@ -0,0 +1,17 @@
+<html>
+  <title>
+    <h1>
+      <b>MINT</b>
+    </h1>
+    The Model Innovation Tool
+  </title>
+
+  <body>
+    <p>You've been added as a collaborator on {{.ModelName}} in MINT.</p>
+
+    <p><a href="{{.Host}}/models/{{.ModelID}}/task-list">Start collaborating now</a></p>

```

Comment:
@OddTomBrooks , could you double check the [FIGMA](https://www.figma.com/file/6tkIhfNfcsKEAkqXo1qvXH/MINT---Read-Only-View?node-id=805%3A73121) for this please? I think it's missing some areas from FIGMA. I think the FONT is different as well

Reply:
I have figured out a good way to reflect Figma style in emails:

- Export Figma frame as PDF
- Open PDF in Adobe Acrobat
- Export PDF as HTML
- Format HTML and change generated fields to directives